lib/modules/manager/maven-lockfile/index.ts

```diff
@@ -0,0 +1,78 @@
+import type { StatusResult } from 'simple-git';
+import upath from 'upath';
+import type { ProgrammingLanguage } from '../../../constants';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecResult } from '../../../util/exec/types';
+import { readLocalFile } from '../../../util/fs';
+import { getRepoStatus } from '../../../util/git';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+
+export const supportsLockFileMaintenance = true;
+export const language: ProgrammingLanguage = 'java';
+
+export async function updateArtifacts({
```

Comment:
this doesn't work as expected. renovate will never call this function for maven dependencies.

this function should be on maven manager. the exec command should be skipped if no lockfiles are found.